db/migrate/20150511185213_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :provider,    null: false
+      t.string :uid,         null: false
+      t.string :nickname,    null: false
```

Comment:
`nickname` does represent the GitHub username in the [OAuth hash](https://github.com/intridea/omniauth/wiki/Auth-Hash-Schema#schema-10-and-later), but I agree it does make more sense to have the username as `login`.